client/state/themes/utils.js

```diff
@@ -4,9 +4,85 @@
  * External dependencies
  */
 import startsWith from 'lodash/startsWith';
+import { omit, omitBy } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { DEFAULT_THEME_QUERY } from './constants';
+
+/**
+ * Constants
+ */
+const REGEXP_SERIALIZED_QUERY = /^((\d+):)?(.*)$/;
 
 export const oldShowcaseUrl = '//wordpress.com/themes/';
 
+/**
+ * Utility
+ */
+
+/**
+ * Returns a normalized themes query, excluding any values which match the
+ * default theme query.
+ *
+ * @param  {Object} query Themes query
+ * @return {Object}       Normalized themes query
+ */
+export function getNormalizedThemesQuery( query ) {
```

Comment:
`get...` suggests that it is a getter of some sort. I am not an expert but in functional style name of function should explain action performed on its args so maybe `normalizeThemesQuery`?